roles/nuage_master/tasks/main.yaml

```diff
@@ -0,0 +1,35 @@
+---
+- name: Install Nuage Kubemon
+  sudo: true
+  yum: name={{ nuage_kubemon_rpm }} state=present
+
+- name: set kubeconfig 
+  sudo: true
+  lineinfile: dest={{ kubemon_yaml }} regexp=^kubeconfig line='kubeconfig{{':'}} {{ admin_config }}'
+
+- name: set openshift admin 
+  sudo: true
+  lineinfile: dest={{ kubemon_yaml }} regexp=^openshiftAdmin line='openshiftAdmin{{':'}} {{ os_admin }}'
+
+- name: Set Openshift master config path
+  sudo: true
+  lineinfile: dest={{ kubemon_yaml }} regexp=^openshiftMasterConfig line='openshiftMasterConfig{{':'}} {{ master_config_yaml }}'
```

Comment:
@detiber agreed. The master config has the cluster's network specification. Our plugin parses the master config to get this information. Beyond that, it doesn't have any business to read the master config. I wasn't sure what the "rules" are to get the cluster configuration - I was being told by @rchopra to not rely on etcd to get the config information as it is an implementation artifact.